Add SIEM and SOAR data retention periods (#2446)
SumoLogic/sumologic-documentation: docs/cloud-soar/overview.md

Automated workflow processes help analysts function at an optimal level and reduce the skills gap that exists from the lack of qualified cybersecurity professionals.

### Comprehensive security portfolio
Cloud SOAR comprises both the Automation Service, which allows our Cloud SIEM to leverage the power of automated playbooks, and the full Cloud SOAR. Cloud SOAR combines automation with case management, among many other capabilities aimed at helping your organization modernize security operations.

## Support and compliance

### Data retention

This section lists the retention period for each type of data generated. 

#### Default retention periods by data type

We automatically delete the following customer data according to the table retention period below, except for those that need to ensure HIPAA compliance.

| Data type | Retention period |
| :-- | :-- |
| Incidents | 2 years |
| Triage | 2 years |
| Entities | 2 years |
| Playbook and action executions | 2 years |

> For those that need to ensure HIPAA compliance, we delete data following the retention periods below. If you need to follow HIPAA compliance, it is important to explicitly communicate this when requesting Cloud SOAR activation.

Reviewer: ```suggestion
For HIPAA-compliant customers, we delete data following the retention periods below. 

:::info
If you need to follow HIPAA compliance, it is important to explicitly communicate this when requesting Cloud SOAR activation.
:::
```

> For clients that need to ensure HIPAA compliance, we delete data following the retention periods below. If you need to follow HIPAA compliance, it is important to explicitly communicate this when requesting Cloud SOAR activation.

Author: ```suggestion
For those that need to ensure HIPAA compliance, we delete data following the retention periods below. If you need to follow HIPAA compliance, it is important to explicitly communicate this when requesting Cloud SOAR activation.
```

Automated workflow processes help analysts function at an optimal level and reduce the skills gap that exists from the lack of qualified cybersecurity professionals.

### Comprehensive security portfolio
Cloud SOAR comprises both the Automation Service, which allows our Cloud SIEM to leverage the power of automated playbooks, and the full Cloud SOAR. Cloud SOAR combines automation with case management, among many other capabilities aimed at helping your organization modernize security operations.

## Support and compliance

### Data retention

This section lists the retention period for each type of data generated. 

#### Default retention periods by data type

We automatically delete the following customer data according to the table retention period below, except for clients that need to ensure HIPAA compliance.

| Data type | Retention period |
| :-- | :-- |
| Incidents | 2 years |
| Triage | 2 years |
| Entities | 2 years |
| Playbook and action executions | 2 years |

> For clients that need to ensure HIPAA compliance, we delete data following the retention periods below. Please keep in mind that if a customer needs to follow HIPAA compliance, it is important to explicitly communicate this when requesting Cloud SOAR activation.

Reviewer: ```suggestion
For clients that need to ensure HIPAA compliance, we delete data following the retention periods below. If you need to follow HIPAA compliance, it is important to explicitly communicate this when requesting Cloud SOAR activation.
```

| Data type | Retention period |
| :-- | :-- |
| Incidents | 7 years |
| Triage | 7 years |
| Entities | 7 years |
| Playbook and action executions | 7 years |

#### Custom retention periods

> Customers can request retention period times different from those declared in the tables above, as long as the retention
> period requested is greater than 1 day yet less than 5000 days.

Reviewer: ```suggestion
You can request retention period times different from those declared in the tables above, as long as the retention period requested is greater than 1 day yet less than 5000 days.
```

### Comprehensive security portfolio

> Cloud SOAR is part of Sumo Logic’s comprehensive security portfolio including security monitoring and analytics and Cloud SIEM, to help your organization modernize security operations.

Reviewer 2: Suggestion:

Cloud SOAR comprises both the Automation Service, which allows our Cloud SIEM to leverage the power of automated playbooks, and the full Cloud SOAR which includes case management among many other capabilities aimed at helping your organization modernize security operations

> Cloud SOAR is part of Sumo Logic’s comprehensive security portfolio including security monitoring and analytics and Cloud SIEM, to help your organization modernize security operations.

Author: @vosorioparis, I used your suggested sentence, and broke it into two:
> Cloud SOAR comprises both the Automation Service, which allows our Cloud SIEM to leverage the power of automated playbooks, and the full Cloud SOAR. Cloud SOAR combines automation with case management, among many other capabilities aimed at helping your organization modernize security operations.

> ## Support and complaince

Reviewer 2: compliance

Author: Good catch! Fixed.

### Data retention

This section lists the retention period for each type of data generated. 

#### Default retention periods by data type

We automatically delete the following customer data according to the table retention period below, except for clients that need to ensure HIPAA compliance.

| Data type | Retention period |
| :-- | :-- |
| Incidents | 2 years |
| Triage | 2 years |
| Entities | 2 years |
| Playbook and action executions | 2 years |

For clients that need to ensure HIPAA compliance, we delete data following the retention periods below. Please keep in mind that if a customer needs to follow HIPAA compliance, it is important to explicitly communicate this when requesting Cloud SOAR activation.

| Data type | Retention period |
| :-- | :-- |
| Incidents | 7 years |
| Triage | 7 years |
| Entities | 7 years |
| Playbook and action executions | 7 years |

> | CSE Records | Records (normalized logs) are stored in the partitions whose names begin with the string `sec_records`. There is one partition for each Record type. <br/><br/>There is no additional charge for storage of Records.| 90 days |

Reviewer 2: Sorry my bad, three last rows belong to CSE, can you please remove them?

Author: I removed the rows "CSE Records", "CSE Signals", and "CSE Insights".
